### PR TITLE
Dockerize biocro

### PR DESCRIPTION
**Pull request by robkooper**
cleanup of #2320

This will dockerize biocro for PEcAn and make it discoverable.

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [X] When possible

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [X] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by robkooper** [comment]
Build I think is ok, just timed out. Both old and develop of R build pass, current R times out.

File: models/biocro/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ----------------------------------------------------------------------
+# BUILD PECAN FOR BIOCRO
+# ----------------------------------------------------------------------
+FROM pecan/models:${IMAGE_VERSION}
```
Comment (kimberlyh66) [inline models/biocro/Dockerfile]: I am not able to build image from this dockerfile. Should there be an IMAGE_VERSION ARG variable set up before this step?

`ARG IMAGE_VERSION='develop'`

Comment (robkooper) [reply models/biocro/Dockerfile]: doh, correct.

**Comment by kimberlyh66** [comment]
Sorry about the self-request review. I am not sure how to remove it.

File: models/biocro/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# this needs to be at the top, what version are we building
+ARG IMAGE_VERSION="latest"
```
Comment (kimberlyh66) [inline models/biocro/Dockerfile]: Should image version be `develop`? I am getting the error : `manifest for pecan/models:latest not found`.

Comment (robkooper) [reply models/biocro/Dockerfile]: You can use `docker build --build-arg IMAGE_VERSION=develop -t pecan/model-biocro .`

Comment (kimberlyh66) [reply models/biocro/Dockerfile]: Ok.

**Pull request merged**